core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronFlowClassifier.java

```diff
@@ -0,0 +1,230 @@
+package org.openstack4j.openstack.networking.domain.ext;
+
+import java.util.List;
+
+import org.openstack4j.model.network.ext.FlowClassifier;
+import org.openstack4j.model.network.ext.builder.FlowClassifierBuilder;
+import org.openstack4j.openstack.common.ListResult;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+/**
+ *
+ *
+ * @author Dmitry Gerenrot
+ *
+ */
+@JsonRootName("flow_classifier")
+public class NeutronFlowClassifier implements FlowClassifier {
+
+	private static final long serialVersionUID = 1L;
+
+	@JsonProperty
+	private String id;
+
+	@JsonProperty
+	private String name;
+
+	@JsonProperty("project_id")
+	private String tenantId;
```

Comment:
use project id for the field name as well. Openstack replaced all references of tenant -> project

Reply:
Agreed. tenant only causes confusion